visallo/visallo
# Remove code that sets a concept type on edges

Testing Instructions:
None

Points of Regression:
None

CHANGELOG
Fixed: The visallo concept type property should not be set on edges
